src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -681,7 +681,7 @@ while bRunning do
 
             end
 
-        elseif param == keys.leftCtrl or param == keys.rightCtrl or param == keys.rightAlt then
+        elseif param == keys.leftCtrl or param == keys.rightCtrl then
```

Comment:
`elseif param == keys.leftCtrl or param == keys.rightCtrl or ( not settings.get("edit.ignore_right_alt") andparam == keys.rightAlt) then`

Reply:
I agree if `edit.ignore_right_alt` is true by default.

Reply:
I don't think that would be best for majority user experience.